docs/book/src/reference/operator-scope.md

```diff
@@ -0,0 +1,269 @@
+# Operator Scope
+
+This section dives into the two crucial concepts of Custom Resource Definitions (CRDs) and Operators, specifically their respective scope and how both impact resource management in your Kubernetes environment.
```

Comment:
```suggestion
```

I think we do not follow this standard in the Kubebuilder doc we do not need this header.